doc/make_doc.in

```diff
@@ -3,32 +3,57 @@
 set -e
 set -o pipefail
 
-abs_top_builddir=@abs_top_builddir@
-abs_top_srcdir=@abs_top_srcdir@
-
-GAP=$abs_top_builddir/bin/gap.sh
+GAP=@abs_top_builddir@/bin/gap.sh
 GAPARGS="-b -m 1g -x 80 -q -r --quitonbreak"
 
 echo "--------------------"
 echo "Building GAP manuals"
 echo "--------------------"
 
 "$GAP" $GAPARGS -A <<EOF
-if LoadPackage("io") = fail then
-  Error("Could not load IO package");
-fi;
-base:="$abs_top_srcdir";;
-books:=["ref", "tut", "changes", "hpc"];;
+base:="@abs_top_srcdir@";;
+books:=["ref", "tut", "changes", "hpc", "dev"];;
```

Comment:
I updated this PR to skip over any missing directories.

But I am not sure why we wouldn't include `doc/dev` with GAP 4.9? Perhaps because it is incomplete and out of date? That would make some sense, I guess; but I hope future versions will be shipped.